kernel/futex.c

```diff
@@ -358,20 +374,45 @@ long myst_syscall_futex(
     int val3)
 {
     long ret = 0;
+    uint32_t bitset = FUTEX_BITSET_MATCH_ANY;
+    int _op = op & ~(FUTEX_CLOCK_REALTIME | FUTEX_PRIVATE);
```

Comment:
we use underscore prefixes to denote static definitions.

Reply:
`_op` renamed.